webrender/src/frame.rs

```diff
@@ -548,8 +535,7 @@ impl Frame {
                                       &iframe_stacking_context,
                                       iframe_clip);
 
-        context.builder.pop_clip_scroll_node();
-        context.builder.pop_clip_scroll_node();
```

Comment:
We no longer push the node to a stack (https://github.com/servo/webrender/pull/978/files#diff-d368cc4d16f067bc1480e7806819a74eL339), so perhaps it would be more accurate to rename that method to add_clip_scroll_node.